exercises/simple-linked-list/src/main/scala/SimpleLinkedList.scala

```diff
@@ -0,0 +1,10 @@
+import scala.reflect.ClassTag
+
+trait SimpleLinkedList[T] {
```

Comment:
I was kinda assuming that the implementers could come up with the companion object themselves by the time they get to this exercise.
Do you think this is too much to expect?